### PR TITLE
Remove remaining CCE identifiers from RHEL derivatives.

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Remove remaining CCE identifier references from RHEL datastream derivatives under the `definition` (OVAL check) tag.

#### Rationale:

- CCE identifiers are bound to RHEL and should not appear in derivatives datastreams

Additional information:
- Values such as:
```
<ns3:reference ref_id="CCE-80657-0" source="CCE"/>
```

### CODE REVIEW

File: ssg/build_derivatives.py [hunk 1]
```diff
@@ -108,6 +108,17 @@ def remove_idents(tree_root, namespace, prod="RHEL"):
                 fix.text = re.sub(r"CCE-[0-9]*-[0-9]*", "", fix.text)
 
 
+def remove_cce_reference(tree_root, namespace, prod="RHEL"):
```
Comment (matejak) [inline ssg/build_derivatives.py]: The `prod` argument is not used in the function, so what about removing it?

Comment (ggbecker) [reply ssg/build_derivatives.py]: Removed it.

**Comment by ggbecker** [comment]
> There is just a minor style nitpick, but besides that, I have verified that the PR works, so it is good to merge.

@matejak  I fixed the style issue. Can you review it again?

**Comment by jan-cerny** [comment]
@ggbecker I'm taking this because @matejak isn't available

**Comment by jan-cerny** [comment]
LGTM

**Pull request merged**